lib/presentation/atoms/check_box_atom.dart

```diff
@@ -7,11 +7,13 @@ class CheckBoxAtom extends StatefulWidget {
     this.text, {
     required this.callback,
     this.initialValue = false,
+    this.trailing,
   });
 
   final String text;
   final bool initialValue;
   final Function(bool) callback;
+  final Widget? trailing;
```

Comment:
Currently it is only an arrow, let's move the arrow icon inside the atom.
Replace getting a widget with getting a callback and show the arrow only if callback were given

Reply:
Sure.